patterns/post-meta.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * Title: Post Meta
+ * Slug: twentytwentythree/post-meta
+ * Categories: post meta
+ * Block Types: core/template-part/post-meta
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"margin":{"top":"var:preset|spacing|70","bottom":"var:preset|spacing|70"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group" style="margin-top:var(--wp--preset--spacing--70);margin-bottom:var(--wp--preset--spacing--70)">
+	<!-- wp:separator {"opacity":"css","align":"wide","className":"is-style-wide"} -->
+	<hr class="wp-block-separator alignwide has-css-opacity is-style-wide"/>
+	<!-- /wp:separator -->
+
+	<!-- wp:columns {"align":"wide","style":{"spacing":{"margin":{"top":"var:preset|spacing|40"},"blockGap":"var:preset|spacing|70"}},"fontSize":"small"} -->
+	<div class="wp-block-columns alignwide has-small-font-size" style="margin-top:var(--wp--preset--spacing--40)">
+		<!-- wp:column {"style":{"spacing":{"blockGap":"0px"}}} -->
+		<div class="wp-block-column">
+			<!-- wp:group {"style":{"spacing":{"blockGap":"1ch"}},"layout":{"type":"flex"}} -->
+			<div class="wp-block-group">
+				<!-- wp:paragraph --><p><?php echo esc_html_x( 'Posted', 'Verb to explain the publication status of a post', 'twentytwentythree' ); ?></p><!-- /wp:paragraph --><!-- wp:post-date {"format":"F j, Y"} /-->
```

Comment:
Does the block break if the date format is made translatable? (untested)

Reply:
No, it seems to be fine! I've just pushed a commit that allows this to be translatable, and I can see the format still working correctly both in the editor and on the front end. I'm also still able to change it from the site editor.